src/github.com/travis-ci/worker/config/config.go

```diff
@@ -10,10 +10,6 @@ import (
 	"github.com/codegangsta/cli"
 )
 
-var (
-	zeroDuration time.Duration
```

Comment:
@meatballhat I'm guessing this was a leftover from some older version of the CLI parsing, but could you verify that this isn't needed anywhere? I did search through the codebase and couldn't find `zeroDuration` anywhere else.